### PR TITLE
fix: [CI-0]: Update comment for getFiles method

### PR DESCRIPTION
**Pull request by hemanthmantri**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/37866)

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_NA_** 
 


            
 [Details](https://app.embold.io/organization/gh/harness/repositories/7752dbb3099d2dba3b5d37f5931a5e4c/pull-request/37866)

File: product/ci/addon/testreports/junit/junit.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ type Junit struct {
 	Log   *zap.SugaredLogger
 }
 
-// getFiles resolves a regex and returns the result
+// Resolves glob pattern for a given path and returns the results
```
Comment (jamie-harness) [inline product/ci/addon/testreports/junit/junit.go:31]: returns list of matched paths

File: product/ci/addon/testreports/junit/junit.go [hunk 2]
```diff
@@ -28,7 +28,7 @@ type Junit struct {
 	Log   *zap.SugaredLogger
 }
 
-// getFiles resolves a regex and returns the result
+// Resolves glob pattern for a given path and returns list of matching paths
```
Comment (rutvijmehta-harness) [inline product/ci/addon/testreports/junit/junit.go:31]: s/Resolves/getFiles resolves

Comment (hemanthmantri) [reply product/ci/addon/testreports/junit/junit.go:31]: I think that's redundant. That's why removed it.

**Review by jamie-harness — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37866)* status: 1. unresolved discussions. 0 of 1 approvals obtained.

**Review by jamie-harness — DISMISSED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37866)* status: 1. unresolved discussions.

**Comment by rutvijmehta-harness** [comment]
*[`product/ci/addon/testreports/junit/junit.go` line 31 at r2](https://reviewable.io/reviews/harness/harness-core/37866#-NCfGbom_8hAF2FMGUoG-r2-31:-NChLhfQ-piHiJMU9U7u:b-vred5t) ([raw file](https://github.com/harness/harness-core/blob/0a4c8a15b10b909d96d44e6a24174817951f1170/product/ci/addon/testreports/junit/junit.go#L31)):*
<details><summary><i>Previously, hemanthmantri (Hemanth Mantri) wrote…</i></summary><blockquote>

I think that's redundant. That's why removed it.
</blockquote></details>

It'll work but it's a Go thing to write the method name followed by what it does.

**Review by jamie-harness — APPROVED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/37866)* status: :shipit: complete! ✓

**Pull request merged**